woocommerce/woocommerce-gutenberg-products-block
# Mini Cart Contents block: implement view switcher in editor

This PR adds a view switcher to the Mini Cart Contents block followed the Cart block method. Note that only switcher and placeholder for empty/filled state for editor were added. The front end of the mini cart content is unchanged.

Fixes #4857




### Screenshots
![image](https://user-images.githubusercontent.com/5423135/144645635-4474b108-3d95-4599-a53a-54c4c40349a0.png)



### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Edit the Mini Cart template part.
2. Select the Mini Cart Contents block.
3. See the switcher in the block toolbar.
4. Try toggling the state, see the content change respectively.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
